src/krux/display.py

```diff
@@ -240,34 +267,60 @@ def clear(self):
         """Clears the display"""
         lcd.clear()
 
+    def draw_touch_button(
+        self, lines, offset_y=0
+    ):  # todo, lcd won't let draw rectangle over 320
```

Comment:
Is this todo still true after your recent changes to `py_lcd`?

Reply:
Rectangle issue was already fixed on this PR, forgot to delete comment.
I'll go through all review items!
Next PR will also have a dedicated method for dynamic touch menu, removed draw_button and cleaned unused variables from `font` and `py_lcd`.
There are still some things to be adapted to touch, such as settings, but I think I should focus on settling all reviewed items and recent changes. Than, before any new code or feature, I could help with documentation, learn about and hopefully also help with tests and simulator.